sopel.py

```diff
@@ -1,6 +1,5 @@
-#!/usr/bin/env python3
-# coding=utf-8
-from __future__ import absolute_import, division, print_function, unicode_literals
+#!/usr/bin/env python
```

Comment:
Is this change wise? Plenty of distros still default to Python 2 if `python3` isn't specified.

Reply:
Not safe, reverted.